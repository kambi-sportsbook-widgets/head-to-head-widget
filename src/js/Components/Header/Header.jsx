import React from 'react'
import PropTypes from 'prop-types'
import styles from './Header.scss'

const Header = ({ title }) => {
  return (
    <div className={styles.general + ' KambiWidget-card-support-text-color'}>
      {title}
    </div>
  )
}

Header.propTypes = {
  title: PropTypes.string.isRequired,
}

export default Header
