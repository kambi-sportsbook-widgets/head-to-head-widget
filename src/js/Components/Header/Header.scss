:local(.general) {
   font-weight: 700;
   font-size: 16px;
   text-align: center;
   padding: 16px 16px 10px;
}
