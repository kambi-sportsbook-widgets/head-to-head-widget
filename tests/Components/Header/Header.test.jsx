/* eslint-env jest */
import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../../src/js/Components/Header/Header';

let renderer;

describe('Header.render()', () => {

   beforeEach(() => {
      renderer = new ReactShallowRenderer();
   });

   it('renders correctly with title', () => {
      expect(renderer.render(
         <Header title="test title" />
      )).toMatchSnapshot();
   });

});
