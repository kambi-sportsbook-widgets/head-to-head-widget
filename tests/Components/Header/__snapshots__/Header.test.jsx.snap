// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.render() renders correctly with title 1`] = `
<div
  className="general KambiWidget-card-support-text-color"
>
  test title
</div>
`;
